analyzer/codechecker_analyzer/cmd/checkers.py

```diff
@@ -255,15 +266,60 @@ def format_guideline(guideline):
         return ' '.join('Related {} rules: {}'.format(g, ', '.join(r))
                         for g, r in guideline.items())
 
+    def guideline_rules_for_checker(checker):
+        """
+        Returns the guideline and rules covered by the given checker. This
+        function returns a dict which maps guideline names to the covered
+        rules: { "sei-cert": ["arr30-c", ...], ... }.
+        """
+        result = defaultdict(list)
+        labels = context.checker_labels.labels_of_checker(checker)
+        guidelines = \
+            context.checker_labels.get_constraint('guideline', 'choice')
+
+        for label in labels:
+            if label[0] in guidelines:
+                result[label[0]].append(label[1])
+
+        return result
+
+    cl = context.checker_labels
+
+    if 'list_label' in args:
+        if args.list_label:
+            rows = cl.get_constraint(args.list_label, 'choice')
+
+            if rows is None:
+                rows = cl.occurring_values(args.list_label)
+                rows = [(row, '') for row in rows]
+            else:
+                rows = rows.items()
+
+            if 'details' in args:
+                header = ['Label', 'Description']
+            else:
+                header = ['Label']
+                rows = [[row[0]] for row in rows]
+        else:
+            header = ['Label']
+            rows = [[label] for label in cl.labels()]
+
+        if args.output_format in ['csv', 'json']:
+            header = list(map(uglify, header))
+
+        print(twodim.to_str(args.output_format, header, rows))
+        return
+
+    # TODO: --profile is a deprecated flag. This section should be removed in
```

Comment:
We should write a warning message if this option is still used to identify that it will be removed in the next release.